shared/js/download/download_formatter.js

```diff
@@ -75,8 +75,7 @@
         }
 
         LazyLoader.load(['shared/js/l10n_date.js'], function onload() {
-          var prettyDate = navigator.mozL10n.DateTimeFormat().fromNow(date);
-          resolve(prettyDate);
+          navigator.mozL10n.DateTimeFormat().relativeDate(date).then(resolve);
```

Comment:
Oh, that's great!  @zbraniecki, sounds like you could use that instead of creating a new `Promise` then.